server/routerlicious/packages/routerlicious-base/src/test/nexus/io.spec.ts

```diff
@@ -338,130 +342,427 @@ describe("Routerlicious", () => {
 				});
 
 				describe("#submitSignal", () => {
```

Comment:
I have mentioned earlier the behavior use case where legitimate client disconnects, and somebody targets it? Is that valid use case?

Any other interesting behavior you can think of?